packages/functions/src/reports/BookedDonationInHospitalReportHandler.ts

```diff
@@ -17,7 +17,7 @@ export default async function (
     throw Error(`User ${callerId} is not an admin`);
   }
 
-  // TODO switch (coordinator.role) && by requested hospital
+  // TODO implement switch (coordinator.role) && by requested hospital
```

Comment:
Haha I meant you need to implement this :P

Why did you chose to add a TODO and not write the code?